Gemfile

```diff
@@ -19,8 +19,8 @@ gem "jekyll", "~> 3.6.2"
 
 # If you have any plugins, put them here!
 group :jekyll_plugins do
-  gem "jekyll-feed", "~> 0.6"
-  gem "github-pages"
+  gem "jekyll-feed"
+  gem "github-pages", "~> 209"
```

Comment:
why this change?

Reply:
`github-pages` and `jekyll` were refusing to update until both packages had minimum versions, because they both depended on each other.

Reply:
Maybe now that they've been updated, we could remove this?  Although I feel like we'd have the same problem again.

Reply:
Nah that makes sense so we can go ahead and leave them